src/main/java/app/MainGameSearchApplication.java

```diff
@@ -16,7 +16,7 @@ public static void main(String[] args) {
         // Initialize dependencies for FetchPlayerStatisticsInteractor
         PlayerStatisticsRepositoryImpl playerStatisticsRepository = new PlayerStatisticsRepositoryImpl(); // Replace with actual implementation
         FetchPlayerStatisticsInteractor fetchStatsBoundary = new FetchPlayerStatisticsInteractor(playerStatisticsRepository);
-        PlayGameController playGameController = new PlayGameController(fetchStatsBoundary);
+        PlayGameController playGameController = PlayGameUseCaseFactory.createController();
 
         // Initialize dependencies for SearchInteractor
         DBSearchDataAccessObject dbSearchDataAccess = new DBSearchDataAccessObject(); // Implements SearchDataAccessInterface
```

Comment:
Consider extracting this initialization logic to a factory or dependency injection framework for better modularity.